tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/Package/VersionUpdateTool.cs

```diff
@@ -0,0 +1,242 @@
+            }
+
+            // Check for package type
+            var packageInfo = await languageService.GetPackageInfo(packagePath, ct);
+            if (packageInfo?.SdkType == SdkType.Management)
+            {
+                // For management-plane packages, execute configured version update script
+                var (configContentType, configValue) = await _specGenSdkConfigHelper.GetConfigurationAsync(sdkRepoRoot, SpecGenSdkConfigType.UpdateVersion);
+                if (configContentType != SpecGenSdkConfigContentType.Unknown && !string.IsNullOrEmpty(configValue))
+                {
+                    logger.LogInformation("Found valid configuration for updating version. Executing configured script...");
+
+                    // Prepare script parameters
+                    var scriptParameters = new Dictionary<string, string>
+                    {
+                        { "SdkRepoPath", sdkRepoRoot },
+                        { "PackagePath", packagePath }
+                    };
+
+                    // Add optional parameters if provided
+                    if (!string.IsNullOrWhiteSpace(releaseType))
+                    {
+                        scriptParameters["ReleaseType"] = releaseType;
+                    }
+
+                    if (!string.IsNullOrWhiteSpace(version))
+                    {
+                        scriptParameters["Version"] = version;
+                    }
+
+                    if (!string.IsNullOrWhiteSpace(releaseDate))
+                    {
+                        scriptParameters["ReleaseDate"] = releaseDate;
+                    }
+                    
+                    // Create and execute process options for the update-version script
+                    var processOptions = _specGenSdkConfigHelper.CreateProcessOptions(configContentType, configValue, sdkRepoRoot, packagePath, scriptParameters);
+                    if (processOptions != null)
+                    {
+                        // Use custom ExecuteProcessAsync that refetches packageInfo after the version update
+                        return await ExecuteProcessAndRefetchPackageInfoAsync(
+                            packagePath,
+                            languageService,
+                            processOptions,
+                            packageInfo,
+                            ct,
+                            $"Version updated" + 
+                                (!string.IsNullOrWhiteSpace(releaseDate) ? $" and release date set to {releaseDate}." : "."),
+                            ["Review the updated version and release date", "Run validation checks"]);
+                    }
+                }
+            }
+
+            // Run default logic to update version
+            logger.LogInformation("Running default logic to update version for the package...");            
+            return await languageService.UpdateVersionAsync(packagePath, version, releaseDate, ct);
```

Comment:
Shouldn't you also pass down releaseType?

Reply:
I would like to keep the args consistent for both the language contract and the script contract.